src/worker/start.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+WORK_DIR=$(dirname "${BASH_SOURCE[0]}")
+
+PROF_METRICS=$1
+
+#shutdown previous instances
+$WORK_DIR/shutdown.sh
+
+# start exporters
+if [ -e "/dev/nvidiactl" ];
+then
+    nohup nv-hostengine </dev/null >/dev/null 2>&1 &
+    if [ $PROF_METRICS = true ];
+    then
+        nohup python3 $WORK_DIR/exporters/nvidia_exporter.py -m </dev/null >/dev/null 2>&1 &
+    else
+        nohup python3 $WORK_DIR/exporters/nvidia_exporter.py  </dev/null >/dev/null 2>&1 &
+    fi
+elif [ -e '/dev/kfd' ];
+then
+    nohup /opt/rocm/rdc/bin/rdcd -u </dev/null >/dev/null 2>&1 &
```

Comment:
Should this full path be hardcoded?

Reply:
This is how it was first coded in ansible. Maybe we can address this in another PR.